devnet/node.yaml

```diff
@@ -103,6 +103,8 @@ spec:
             - ws://eth-devnet:8545
             - --baseRPC
             - ws://eth-devnet:8545
+            - --scrollRPC
+            - ws://eth-devnet:8545
```

Comment:
Won't this fail the guardian start due to [this code](https://github.com/wormhole-foundation/wormhole/pull/3429/files#diff-7f43e1a5bbb6083b5e470710e19e804d778f4a411b23fdbef1b2955f56b67bbdR637-R639) added in this PR? Seems like a contract needs to be specified and I'm clearly missing some magic here.

Reply:
No, this is not a problem. In tilt (unsafeDevMode) the contract address for all the EVM chains are set to the one deployed locally. See line node.go, line 498.